etc/conf-standalone.template

```diff
@@ -1,7 +1,7 @@
 /////////////////////////////////////////////////////////////////////////////////////////
 //
 // S T A N D A L O N E
-//
+// REVISION_TAG=1.0
```

Comment:
manage configuration upgrades with docker -> put the rigth information on the container startups when the actual configuration file saved in the volume dosent match with the latest version of warp10 platform